ios/Classes/core/PMManager.m

```diff
@@ -479,6 +479,12 @@ - (void)getFullSizeFileWithId:(NSString *)id isOrigin:(BOOL)isOrigin subtype:(in
                 return;
             }
         }
+        if (@available(macOS 14.0, *)) {
+            if (asset.isLivePhoto && (subtype & PHAssetMediaSubtypePhotoLive) == PHAssetMediaSubtypePhotoLive) {
+                [self fetchLivePhotosFile:asset handler:handler progressHandler:progressHandler];
+                return;
+            }
+        }
```

Comment:
When I was looking at the problem with the file name, I found that macOS and iOS have different logic about getting livePhoto, so I add the code.